plugins/@grouparoo/airtable/src/lib/client/client.ts

```diff
@@ -0,0 +1,188 @@
+import Airtable, { FieldSet, Record, RecordData, Records } from "airtable";
```

Comment:
This is a very well typed client!
I like how you extended the `airbase` client with the missing methods you needed